Neos.Flow/Configuration/Settings.Object.yaml

```
@@ -22,4 +22,32 @@ Neos:
       # reflected by default. You can however exclude specific (or all) classes from Flow packages
       # by specifying corresponding regular expressions that don't match classes to exclude.
       #
-      includeClasses: []
+      includeClasses: [ ]
+
+      dependencyInjection:
+
+        # Explicitly exclude classes from constructor autowiring.
+        #
+        # This setting accepts an array of fully qualified class names, each class name
+        # being a regular expression.
```

It's not actually FQN IMO, maybe something like:
```suggestion
        # This setting accepts an array of regular expressions, each matching one or more fully
        # qualified class names.
```